src/components/App.js

```diff
@@ -2,14 +2,32 @@ import React from 'react';
 import './App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import Calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <div className="App">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  HandleClick(buttonName) {
+    const data = Calculate(this.state, buttonName);
```

Comment:
Calculate is a function not a class change it to a lower case here and on the calculate.js file.
From `Calculate` to `calculate`